tools/build.js

```diff
@@ -55,10 +55,7 @@ const commonConfig = {
         }),
         babel({
             exclude: 'node_modules/**',
-            plugins: [
-                'external-helpers',
```

Comment:
removed because this happen automatically now
```
Using "external-helpers" plugin with rollup-plugin-babel is deprecated, as it now automatically deduplicates your Babel helpers.
```